journals/apps/api/v1/views.py

```diff
@@ -62,3 +67,20 @@ def _revoke_access(self, order_number):
                     access_record.journal,
                     access_record.order_number)
         return HttpResponse()
+
+
+class UserPageVisitView(generics.ListCreateAPIView):
+    """API view for UserPageVisit model"""
+    serializer_class = UserPageVisitSerializer
+    filter_backends = (SearchFilter,)
+    search_fields = ('page__id',)
```

Comment:
For consistency would be better to use 
```
filter_backends = (DjangoFilterBackend, )
filter_class = UserPageVisitFilter
```
instead of SearchFilter and search_fields. You'd need to define your UserPageVisitFilter class. See what @BessieSteinberg did in her PR https://github.com/edx/journals/pull/67

This way your filters are just query string params like /user/id/pagevisits?page_id=1 with no weird search=xxx querystring